Scripts/prepare.prod.sh

```
@@ -39,6 +41,15 @@ for framework_path in $FRAMEWORK_PATTERN; do
         fi
         echo "Target name: $target_name"
 
+        APPLE_DEVELOPMENT_SIGNING_IDENTITY="$(security find-identity -v -p codesigning | grep "Apple Development:" | head -n 1 | awk '{print $2}')"
+        if [ -z "$APPLE_DEVELOPMENT_SIGNING_IDENTITY" ]; then
+            echo "Error: No Apple Development signing identity found in the keychain. To check available, try running: security find-identity -v -p codesigning | grep \"Apple Development:\""
+            exit 1
+        fi
+        codesign --timestamp -v -f --sign "$APPLE_DEVELOPMENT_SIGNING_IDENTITY" "$framework_path"
+        echo $(codesign -dv "$framework_path")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)